### PR TITLE
run generator for updates on several fields

### PR DESCRIPTION
**Pull request by sanderland**
also a quick test for dataset id on asset since someone complained about it, but seems to be there already

### CODE REVIEW

**Comment by polomani** [comment]
```
.data_set_id.set(123),
```
it just this basically? 
the rest we have in https://github.com/cognitedata/cognite-sdk-python/pull/673

**Comment by sanderland** [comment]
@polomani  asset_external_ids on files

**Comment by sanderland** [comment]
and yes, this is largely @easyCheesyPotato's fault for his fake bug report :D

**Comment by polomani** [comment]
hi, can you rebase it?

**Comment by sanderland** [comment]
I see it's basically just a missing docstring after stopping interaction mid-review, waiting a week, having me review someone else adding these changes instead, and merging @polomani

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ Changes are grouped as follows
 - Separate read/write fields on data classes
 
 ## Unreleased
+### Added
+- asset_external_ids on filtering in files.
```
Comment (polomani) [inline CHANGELOG.md:24]:
```suggestion
```
not here anymore?

**Pull request merged**